### PR TITLE
Add like table to group dash.

### PR DESCRIPTION
**Pull request by eschlange**
* Add like table to group dash view.
* Add link to like table at top of dash view.

[Finished #90054316]

### CODE REVIEW

File: app/helpers/think_feel_do_engine/coach/group_dashboard_helper.rb [hunk 1]
```diff
@@ -108,6 +108,10 @@ def membership_comments(membership)
         SocialNetworking::Comment.where(participant: membership.participant)
       end
 
+      def membership_likes(membership)
+        SocialNetworking::Like.where(participant: membership.participant)
```
Comment (ericcf) [inline app/helpers/think_feel_do_engine/coach/group_dashboard_helper.rb]: Would this be faster if you used `where(participant_id: membership.participant_id)` instead, or is it about the same?

**Comment by ericcf** [comment]
@eschlange is there a feature spec somewhere to confirm that the likes table is included?

File: app/helpers/think_feel_do_engine/coach/group_dashboard_helper.rb (hunk 1 above)
Comment (eschlange) [inline app/helpers/think_feel_do_engine/coach/group_dashboard_helper.rb]: Locally, its virtually the same. It is averaging a few milliseconds apart.

**Comment by eschlange** [comment]
@ericcf The last push includes a feature spec to check for the existence of the likes container.

**Comment by ericcf** [comment]
@eschlange cool, thanks! Will merge unless RuboCop rears his ugly head. :cop:

**Pull request merged**